manual/manual/cmds/Makefile

```diff
@@ -12,9 +12,14 @@ TRANSF=$(SET_LD_PATH) $(OCAMLRUN) ../../tools/transf
 TEXQUOTE=../../tools/texquote2
 FORMAT=../../tools/format-intf
 
-WITH_TRANSF= ocamldoc.tex top.tex intf-c.tex flambda.tex spacetime.tex \
+CAMLLATEX= $(OCAMLRUN) ../../tools/caml-tex2 -caml "TERM=norepeat $(OCAML)" \
+-n 80 -v false
```

Comment:
Does this not need an update to take 482e8a8b52ff9fb8e8f8aecb17264f314b9a37f0 into account?

Reply:
You are totally right, I had forgotten this point. I should try to make my test environment sensitive to this problem. Fixed.